etc/nebula-metad.conf.production

```diff
@@ -33,3 +33,7 @@
 ########## storage ##########
 # Root data path, here should be only single path for metad
 --data_path=data/meta
+
+############## rocksdb Options ##############
+--rocksdb_disable_wal=false
```

Comment:
Why not enable for storage?I think there is secure problem to loss data without replica (or with replica but all node crash).